core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -319,7 +319,18 @@ private[spark] class AppStatusStore(
           toValues(_.shuffleFetchWaitTime),
           toValues(_.shuffleRemoteBytesRead),
           toValues(_.shuffleRemoteBytesReadToDisk),
-          toValues(_.shuffleTotalBlocksFetched)),
+          toValues(_.shuffleTotalBlocksFetched),
+          toValues(_.shuffleRemoteReqsDuration),
+          new v1.ShufflePushReadMetricDistributions(
+            toValues(_.shuffleCorruptMergedBlockChunks),
+            toValues(_.shuffleFallbackCount),
```

Comment:
`shuffleFallbackCount` and `ShufflePushReadMetricDistributions.fallbackCount` needs to be renamed as well to be consistent.

Please do make sure the other renames we did are consistent throughout as well - I will go over it once more later, but would be great if you can take a pass as well @thejdeep . Thx